jmetal-core/src/main/java/org/uma/jmetal/util/experiment/component/GenerateWilcoxonTestTablesWithR.java

```diff
@@ -67,14 +68,14 @@ private void printHeaderLatexCommands(String rFileName, String latexFileName) th
     String dataDirectory = experiment.getExperimentBaseDirectory() + "/data";
     os.write("resultDirectory<-\"" + dataDirectory + "\"" + "\n");
     output = "latexHeader <- function() {" + "\n" +
-        "  write(\"\\\\documentclass{article}\", \"" + latexFileName + "\", append=TRUE)" + "\n" +
-        "  write(\"\\\\title{StandardStudy}\", \"" + latexFileName + "\", append=TRUE)" + "\n" +
-        "  write(\"\\\\usepackage{amssymb}\", \"" + latexFileName + "\", append=TRUE)" + "\n" +
-        "  write(\"\\\\author{A.J.Nebro}\", \"" + latexFileName + "\", append=TRUE)" + "\n" +
-        "  write(\"\\\\begin{document}\", \"" + latexFileName + "\", append=TRUE)" + "\n" +
-        "  write(\"\\\\maketitle\", \"" + latexFileName + "\", append=TRUE)" + "\n" +
-        "  write(\"\\\\section{Tables}\", \"" + latexFileName + "\", append=TRUE)" + "\n" +
-        "  write(\"\\\\\", \"" + latexFileName + "\", append=TRUE)" + "\n" + "}" + "\n";
+        "  write(\"\\\\documentclass{article}\", \"" + latexFileName + APPEND_STRING +
+        "  write(\"\\\\title{StandardStudy}\", \"" + latexFileName + APPEND_STRING +
+        "  write(\"\\\\usepackage{amssymb}\", \"" + latexFileName + APPEND_STRING +
+        "  write(\"\\\\author{A.J.Nebro}\", \"" + latexFileName + APPEND_STRING +
+        "  write(\"\\\\begin{document}\", \"" + latexFileName + APPEND_STRING +
+        "  write(\"\\\\maketitle\", \"" + latexFileName + APPEND_STRING +
+        "  write(\"\\\\section{Tables}\", \"" + latexFileName + APPEND_STRING +
+        "  write(\"\\\\\", \"" + latexFileName + APPEND_STRING + "}" + "\n";
```

Comment:
This is not only the `APPEND_STRING` which is repetitive. Also the `latexFileName`. Surely a factory instantiated with `latexFileName` as a constructor argument would be interesting here to simplify the code.